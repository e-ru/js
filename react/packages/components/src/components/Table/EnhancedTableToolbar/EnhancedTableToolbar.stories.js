import React from "react";
import { storiesOf } from "@storybook/react";

import EnhancedTableToolbar from "./EnhancedTableToolbar";

storiesOf("EnhancedTableToolbar", module).add("render", () => <EnhancedTableToolbar title="main" numSelected={0} />);
