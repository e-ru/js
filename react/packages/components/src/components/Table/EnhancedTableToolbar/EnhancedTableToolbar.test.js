import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import EnhancedTableToolbar from "./EnhancedTableToolbar";

configure({ adapter: new Adapter() });

function setup() {
  const props = {
    title: "test_title",
    numSelected: 0,
  };

  const wrapper = shallow(<EnhancedTableToolbar {...props} />);

  return {
    wrapper,
  };
}

describe("<EnhancedTableToolbar />", () => {
  test("renders", () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
