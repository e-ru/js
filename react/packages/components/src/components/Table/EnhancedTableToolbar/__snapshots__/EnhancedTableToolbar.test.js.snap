// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EnhancedTableToolbar /> renders 1`] = `ShallowWrapper {}`;
